tools/devlog/log_to_file.go

```diff
@@ -0,0 +1,37 @@
+// A logger that writes to a tmp file. For use in development only
+// (presumably for HUD development).
+
+package devlog
+
+import (
+	"fmt"
+	"os"
+	"time"
+)
+
+var logfileBaseName = "/tmp/tilt-log-%s"
+var theLogfile *os.File // THERE CAN ONLY BE ONE!
+
+func logFileName() string {
+	now := time.Now()
+	tStr := now.Format("2006-01-02T15.04.05")
```

Comment:
I'd vote for just using a constant filename (no timestamp) and opening the file for append, so that we get one file and easy tab completion instead of potentially hundreds of files and annoying tab completion conflicts